### DISCUSSION TITLE
How does use C++ SDK to connect Minio Server configured with self-signed certificates in Windows

### DISCUSSION THREAD

**Discussion by binnetwork** [minio/minio#14161] [Q&A] [answered]
Hi, as the title says, I would like to know how to use the C++ SDK to connect to minio Server that operates on self-signed certificates, I can't find the relevant documentation about it. May anyone show me the point. Thanks in advanced.

**Comment by harshavardhana** [comment, marked as answer]
> Hi, as the title says, I would like to know how to use the C++ SDK to connect to minio Server that operates on self-signed certificates, I can't find the relevant documentation about it. May anyone show me the point. Thanks in advanced.

Trust your self signed cert on the system and use it.

**Reply by binnetwork** [reply]
Can you trust it through the SDK without having to install the trust certificate on the client

**Reply by harshavardhana** [reply]
> Can you trust it through the SDK without having to install the trust certificate on the client

which C++ SDK are you talking about? @binnetwork

**Reply by binnetwork** [reply]
https://github.com/aws/aws-sdk-cpp，The c++ SDK for s3 is used
